css/styles.css

```diff
@@ -47,62 +44,276 @@ nav {
+  margin-left: 27px;
+  margin-right: 37px;
+  margin-top: 10px;
   display: flex;
-  align-items: center;
   justify-content: space-between;
+  align-items: center;
 }
 
+/* Section Headline */
 .headline {
   margin: 20px, 0, 20px, 0;
-  font-family: Poppins, sans-serif;
   color: #172b4d;
-  width: 375px;
+  width: 100%;
   display: flex;
   flex-direction: column;
 }
 
 .logo-txt {
-  font-family: Poppins, sans-serif;
   color: #6070ff;
   font-weight: bold;
   font-size: 18px;
 }
 
 .name {
   width: 85%;
-  margin-top: 20px;
+  margin: 20px 20px 0 24px;
+  color: #172b4d;
 }
 
 .desc {
-  margin-top: 14px;
+  margin: 16px 10px 20px 24px;
   font-weight: 400;
   width: 85%;
-  line-height: 1.5;
-  text-align: justify;
+  color: #344563;
+  line-height: 1.5em;
+}
+
+.connect {
+  width: 85%;
+  margin: 0 20px 15px 24px;
+  color: #7f8cff;
+  font-weight: 500;
 }
 
 .container-3 {
   display: flex;
+  margin: 0 20px 20px 24px;
 }
 
-.connect {
-  margin-top: 20px;
+/* Section Works */
+.works {
+  width: 100%;
+}
+
+.grid-1 {
+  width: 90%;
+  margin-left: auto;
+  margin-right: auto;
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: 640px 700px 640px 700px;
+  gap: 25px;
+}
+
+.work1 {
+  background-color: #fff;
+  border: solid 1px #dfe1e6;
+  border-radius: 16px;
+  margin-top: 30px;
+  margin-bottom: 30px;
+}
+
+.desc2 {
+  font-size: 16px;
+  font-weight: 500;
+  color: #344563;
+  line-height: 1.5em;
+  margin-left: 5%;
+  margin-right: auto;
+  margin-bottom: 15px;
+}
+
+.work1-img {
+  display: block;
+  width: 90%;
+  height: auto;
+  margin-top: 15px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.work-text1 {
+  margin-top: 8px;
+  font-size: 32px;
+  color: #172b4d;
+  width: 70%;
+  margin-left: 5%;
   margin-bottom: 10px;
-  color: #7f8cff;
 }
 
-.linkedin.m.github.sign {
+.canopy {
+  color: #344563;
+  font-size: 12px;
+  font-weight: 600;
+  margin: 0;
+  padding: 0;
+}
+
+.bed,
+.year {
+  color: #7a869a;
+  font-size: 12px;
+  font-weight: 600;
+}
+
+.dot {
+  margin: 0;
+  padding: 0;
+  width: 60%;
+  height: auto;
+}
+
+.stack {
+  font-size: 12px;
+  color: #6070ff;
+  background-color: #ebebff;
+  padding: 4px 12px 4px 12px;
+  border-radius: 4px;
+}
+
+.flex1,
+.flex2 {
+  margin-left: 5%;
+  margin-right: auto;
+  display: flex;
+  align-items: center;
+  margin-bottom: 15px;
+}
+
+.see-project {
+  font-size: 17px;
+  color: #396df2;
+  border: solid 1px #396df2;
+  padding: 8px;
+  border-radius: 4px;
+  margin-left: 5%;
+  margin-right: auto;
+}
+
+.see-project:hover {
+  background-color: #396df2;
+  color: #fff;
+  font-size: 17px;
+  border: solid 1px #396df2;
+  padding: 8px;
+  border-radius: 4px;
+  margin-left: 5%;
+  margin-right: auto;
+}
+
+/* Section About Myself */
+.about_myself {
+  margin: 20px, 0, 20px, 0;
+  color: #172b4d;
+  display: flex;
+  flex-direction: column;
+}
+
+.about {
+  width: 90%;
+  color: #172b4d;
+  font-size: 40px;
+  margin-left: 24px;
+}
```

Comment:
Kindly adjust the skills section to match that of the design. The arrow on the design does not have a background and is a little less bold. Also, the space between the uncollapsed sections is lesser.

Your design:
![image](https://user-images.githubusercontent.com/20554583/229000300-6433a0d7-cd31-4101-a0ec-b9eb26935c13.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/229000561-8b3593ae-e53a-43ea-8286-46d30d0bebfd.png)

Reply:
Good day, Emmanuel: Can you tell me how to eliminate the background of the arrows (in the sections of languages, frameworks & skills)? When I copy them from Figma, they have some background (gray) that I can't take away. Is there a way to copy only the lines of the arrows? Tank you very much.

Reply:
[emmiiorji](https://github.com/emmiiorji)  Good day, Emmanuel: Can you tell me how to eliminate the background of the arrows (in the sections of languages, frameworks & skills)? When I copy them from Figma, they have some background (gray) that I can't take away. Is there a way to copy only the lines of the arrows? Tank you very much.

Reply:
Hi @jcmunav63

You may kindly check out [font awesome](https://fontawesome.com/search?q=down%20arrow&o=r) for similar icons.